pypeit/reduce.py

```diff
@@ -1079,7 +1113,9 @@ def find_objects_pypeline(self, image, std_trace=None,
 
         # Masking options
         if self.par['reduce']['skysub']['mask_by_boxcar']:
-            boxcar_rad_skymask = self.par['reduce']['extraction']['boxcar_radius'] / self.get_platescale(None),
+            bin_spec, bin_spat = parse.parse_binning(self.binning)
+            plate_scale = self.get_platescale(None)*bin_spat
```

Comment:
`self.get_platescale(None)` looks weird.  This goes back to my request to make the optional arguments keywords instead of requiring a dummy argument.